js/ui/editor/editor.js

```diff
@@ -203,11 +217,13 @@ var Editor = Widget.inherit({
     },
 
     _renderValidationState: function() {
-        var isValid = this.option("isValid"),
-            validationError = this.option("validationError"),
+        const isValid = this.option("isValid") && this.option("validationStatus") !== "invalid",
             validationMessageMode = this.option("validationMessageMode"),
             $element = this.$element();
-
+        let validationErrors = this.option("validationErrors");
```

Comment:
```suggestion
        const validationError = this.option("validationError");
        const validationErrors = this.option("validationErrors") || validationError ? [validationError] : null;
```
and remove L224-L226

Reply:
validationErrors is a new option. May not be synchronize after with validationError. 
validationError is going to be deprecated. I do not think that we should remove  L224-L226 now. Only after removing validationError from code after the deprecation.